### ISSUE TITLE
Game crashes when entering multiplayer server

### ISSUE THREAD

**Issue by ruggbean** [anegostudios/VintageStory-Issues#2229]
**Game Version:** 1.17.5
**Platform:** Windows 
**Modded:** Yes (Carry Capacity, Zoom Button, Step Up, More Candles, Better Backpacks, Stackables)
**SP/MP:** Multiplayer

### Description
The server is crashing almost as soon as I log in. This crash was not an issue in 1.17.3 or 1.17.4, only in 1.17.5

### How to reproduce
1. Download 1.17.5
2. Run multiplayer server
3. Have multiplayer server active for approximately 5 seconds.
4. Issue is not replicatable in single-player

### Expected behavior
N/A

### Screenshots
![Capture](https://user-images.githubusercontent.com/109896152/193698337-7280c7a8-200b-4145-a39e-04d595248ee9.PNG)

### Logs
((https://gist.github.com/ruggbean/1cc692aaf26fc199b91b34f413b7d94b))

**Comment by wasakoki** [comment]
I'm getting this issue too on my vanilla multiplayer server. it appeared only after updating from 1.17.4 to 1.17.5
Only I'm having this issue on my server, my friend (who is in a different location in the world) has no issues connecting. according to the logs my friend clearly doesnt have a condenser at their base :p 
### Logs
```
3.10.2022 21:20:00 [Fatal] Caught unhandled exception in thread 'networkproc'. Exiting game.
3.10.2022 21:20:00 [Fatal] System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.GameContent.BlockEntityCondenser.genBucketMesh() in C:\Users\Tyron\Documents\vintagestory\game\VSSurvivalMod\BlockEntity\BlockEntityCondenser.cs:line 149
   at Vintagestory.GameContent.BlockEntityCondenser.FromTreeAttributes(ITreeAttribute tree, IWorldAccessor worldForResolving) in C:\Users\Tyron\Documents\vintagestory\game\VSSurvivalMod\BlockEntity\BlockEntityCondenser.cs:line 293
   at Vintagestory.Client.NoObf.ClientChunk.PreLoadBlockEntitiesFromPacket(Packet_BlockEntity[] blockEntities, Int32 blockEntitiesCount, ClientMain game) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Model\ClientChunk.cs:line 359
   at Vintagestory.Client.NoObf.ClientWorldMap.LoadChunkFromPacket(Packet_ServerChunk p) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientWorldMap.cs:line 657
   at Vintagestory.Client.NoObf.SystemNetworkProcess.ProcessInBackground(Packet_Server packet) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\NetworkProcess.cs:line 218
   at Vintagestory.Client.NoObf.SystemNetworkProcess.TryReadPacket(Byte[] data, Int32 dataLength) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\NetworkProcess.cs:line 152
   at Vintagestory.Client.NoObf.SystemNetworkProcess.NetworkProcess() in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\NetworkProcess.cs:line 130
   at Vintagestory.Client.NoObf.ClientThread.Update() in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientThread.cs:line 80
   at Vintagestory.Client.NoObf.ClientThread.Process() in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientThread.cs:line 34
3.10.2022 21:20:11 [Notification] Finished fully loading sounds (async)
3.10.2022 21:20:34 [Notification] Destroying game session, waiting up to 200ms for client threads to exit
3.10.2022 21:20:34 [Notification] Exiting current game to main menu, reason: client thread crash
3.10.2022 21:20:37 [Notification] Window was resized to 0 0? Window probably got minimized. Will not rebuild frame buffers
3.10.2022 21:20:38 [Warning] Texture with texture id 253 is leaking memory, missing call to Dispose. Set env var TEXTURE_DEBUG_DISPOSE to get allocation trace.

```

**Comment by IvanStamenov** [comment]
The same crash is observed in single player as well.

**Comment by bluelightning32** [comment]
I hit the same crash on 1.17.5. I tried removing all mods, but the client still crashes. Using a 1.17.4 client, I can connect to the 1.17.5 server without a crash.

**Comment by Elywejnak** [comment]
Same issue on our server, I'll add logs from client, but the exception (on the client) seems to be the same one. 

**Logs**
[client-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/9703800/client-main.txt)

**Comment by Craluminum2413** [comment]
Fixed in [1.17.6](https://www.vintagestory.at/forums/topic/10247-stabler-de-jank-and-game-juice-update-1175-1176/#comment-41928:~:text=fix%20a%20client%20crash%20related%20to%20condensers)
